Assets/MixedRealityToolkit.SDK/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -0,0 +1,390 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [Serializable]
+    public struct StabilizationPlaneOverride
+    {
+        public Vector3 Center;
+        public Vector3 Normal;
+    }
+
+    /// <summary>
+    /// StabilizationPlaneModifier handles the setting of the stabilization plane in several ways.
+    /// Using StabilizationPlaneModifier with override DepthLSR which is automatically enabled via depth buffer sharing
+    /// StabilizationPlaneModifier is recommended for HoloLens 1, can be used for HoloLens 2, and does a no op for WMR
+    /// </summary>
+    public class StabilizationPlaneModifier : MonoBehaviour
+    {
+        [System.Serializable]
+        public enum StabilizationPlaneMode
+        {
+            Fixed,
```

Comment:
>Fixed [](start = 12, length = 5)

[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum)

(In particular: DO provide a value of zero on simple enums.)